web/src/message_fetch.ts

```diff
@@ -715,6 +715,39 @@ export function initialize(finished_initial_fetch: () => void): void {
         } catch (error) {
             blueslip.error("Error in recent_view_ui.process_messages", undefined, error);
         }
+
+        // Since we backfill a lot more messages here compared to rendered message list,
+        // we can try populating them if we can do so locally.
+        for (const msg_list of message_lists.all_rendered_message_lists()) {
```

Comment:
Should this be all rendered lists, or all current `MessageListData` objects?

Reply:
Yeah, updated.